AAP-14288 Added an admonition about creating a different name for EDA controller

jself-sudoku opened the pull request:
Added "Important" admonition to the Deploying the Red Hat Ansible Automation Platform operator on OpenShift Container Platform, Chapter 6, informing customers that they should give their EDA controller a different name if they are deploying it in the same namespace as other AAP  components. 

(Note to ask Christian Adams: Do we inform the customer where they should do this renaming? It doesn't seem to be in the contents of Chapter 6 where we talk about deploying EDA on OCP. I might be missing something. Please advise.)

rooftopcellist commented:
@jself-sudoku In between steps 3 and 4 on [this doc page](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/deploying_the_red_hat_ansible_automation_platform_operator_on_openshift_container_platform/deploy-eda-controller-on-aap-operator-ocp), I think we should add a step saying to specify the name of the deployment. We can add that warning as part of that step, or as a warning block, whichever reads better works for me. 

Here's some potential text for Step 3.5:

```
In the "Name" field, specify what you would like your new {EDAcontroller} deployment to be named. All EDA kubernetes resources will created using this name. Note that if you have other {PlatformNameShort} components installed in your current {OCPShort} namespace, ensure that you provide a unique name for your {EDAcontroller} when you create your {EDAName} custom resource. Otherwise, naming conflicts can occur and impact {EDAcontroller} deployment.
```

jself-sudoku commented:
@rooftopcellist - thank you for responding my question about updating the instructions to add a step for naming EDA controller. The information we discussed has been added to the content. I added a step 4 and moved the "Important" admonition from Chapter 6 abstract to the procedural page after step 4. I also tweaked information in step 5 that points the customer to the automation controller URL.

rooftopcellist commented:
@jself-sudoku looks good!

dcdacosta reviewed (approved):
LGTM!
